catalogs/catalog-jdbc-doris/src/main/java/com/datastrato/gravitino/catalog/doris/operation/DorisDatabaseOperations.java

```diff
@@ -4,28 +4,111 @@
  */
 package com.datastrato.gravitino.catalog.doris.operation;
 
+import com.datastrato.gravitino.catalog.doris.utils.DorisUtils;
 import com.datastrato.gravitino.catalog.jdbc.JdbcSchema;
 import com.datastrato.gravitino.catalog.jdbc.operation.JdbcDatabaseOperations;
 import com.datastrato.gravitino.exceptions.NoSuchSchemaException;
+import com.datastrato.gravitino.exceptions.NoSuchTableException;
+import com.datastrato.gravitino.meta.AuditInfo;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.Collections;
+import java.util.List;
 import java.util.Map;
+import org.apache.commons.lang3.StringUtils;
 
 /** Database operations for Doris. */
 public class DorisDatabaseOperations extends JdbcDatabaseOperations {
-  // TODO: add implementation for doris catalog
 
   @Override
-  public JdbcSchema load(String databaseName) throws NoSuchSchemaException {
-    throw new UnsupportedOperationException();
+  public String generateCreateDatabaseSql(
+      String databaseName, String comment, Map<String, String> properties) {
+    if (StringUtils.isNotEmpty(comment)) {
+      throw new UnsupportedOperationException(
+          "Doris doesn't support set database comment: " + comment);
```

Comment:
Since Doris supports database properties, should we store the comment in the properties instead of throwing an exception?

Reply:
BTW, the `gravitino.identifier` was stored in the comment, you should extract it and put it into the properties.

Reply:
Got it, I will update to support comment and gravitino.identifier

Reply:
@mchades I had update code, store comment in database properties.

In my opinion, it is sufficient to store comments in the attribute with `gravitino.identifier`. There is no need to extract it. Do you have any suggestions?